core/services/chainlink/relayer_factory.go

```diff
@@ -67,6 +67,9 @@ func (r *RelayerFactory) NewEVM(config EVMFactoryConfig) (map[types.RelayID]evmr
 	}
 
 	if cmdName := env.EVMPlugin.Cmd.Get(); cmdName != "" {
+		if config.GenEthClient != nil {
+			return nil, errors.New("GenEthClient override is not supported in the loopp mode")
+		}
```

Comment:
Let's do the same for each of these: https://github.com/smartcontractkit/chainlink-evm/blob/develop/pkg/chains/legacyevm/chain.go#L150-L156